### ISSUE TITLE
Spotify playlist stops after a few songs on DLNA speakers

### ISSUE THREAD

**Issue by byronward85** [music-assistant/support#3280]
### What version of Music Assistant has the issue?

2.3.3

### What version of the Home Assistant Integration have you got installed?

2024.12.0

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

I've spent all day searching the internet and messing about with this one...

When playing spotify playlists, they work for the first few songs then just stop. I'm using DLNA players. 

### How to reproduce

Play a spotify playlist through DLNA speakers. I've tried individual and groups which have been assigned in MA. Same issue

### Music Providers

Spotify

### Player Providers

DLNA

### Full log output

[music-assistant (2).log](https://github.com/user-attachments/files/18049851/music-assistant.2.log)

### Additional information

I've tried various options in 'core' but issue not resolved. 

### What version of Home Assistant Core are your running

2024.12.1

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Raspberry Pi

**Comment by OzGav** [comment]
I assume you have tried all the things mentioned in https://music-assistant.io/player-support/dlna/

**Comment by byronward85** [comment]
Yes.. although I dont have an option for 'QUEUE FLOW MODE' in the player settings, be that individual players or groups. I cant find it anywhere?

I've tried all three available HTML profiles for sending audio (chunked, no content length and forced content length) and all have the same issue.

Its very strange as it works perfectly for about 30 mins then it just stops without any errors in the logs.

I'll try disable 'Enfore (lossy) mp3 stream' and see if that has any affect.

Any advice or recommendations highly appreciated

Thanks

**Comment by byronward85** [comment]
I didnt realise my speakers supported Airplay and they work great so I'll use that instead of DLNA

Thanks

**Comment by OzGav** [comment]
If you are happy with AirPlay do you to close this?

**Comment by byronward85** [comment]
Yeah its closed. Airplay seems to be fine
